src/posts/posts.service.ts

```diff
@@ -96,12 +102,12 @@ export class PostsService {
   async createOptionGroup(
     postid: string,
     groupsCreationDto: OptionsGroupCreationDto,
-    userId: number,
+    userId: string,
```

Comment:
why? didn't we align on simple numbers 1,2, 3.. 6?

Reply:
I aligned with front-end team to use uuid instead. I will send them statically craeted uuids for 4 users.
I made this change to prevent changing more code in future updates. It is frustrating to change all these files again. Also it will not affect the front-end team by any means.

Reply:
it is ok. they can map the uuids to mre easy to read/write numbers